gpflow/inducing_variables/multioutput/inducing_variables.py

```diff
@@ -44,7 +44,7 @@ class FallbackSharedIndependentInducingVariables(MultioutputInducingVariables):
     only requiring the following covariances:
      - Kuu: [L, M, M],
      - Kuf: [L, M, N, P].
-    In `mo_conditionals.py` we define a conditional() implementation for this
+    In `multioutput/conditionals.py` we define a conditional() implementation for this
```

Comment:
should this clarify that this is in the `conditionals` subdirectory?
```suggestion
    In `gpflow/conditionals/multioutput/conditionals.py` we define a conditional() implementation for this
```
or
```suggestion
    In `gpflow.conditionals.multioutput` we define a conditional() implementation for this
```